File: grpc/grpc-client-annotation-processor/src/main/java/ru/tinkoff/grpc/client/annotation/processor/GrpcClassNames.java
package ru.tinkoff.grpc.client.annotation.processor;

import com.squareup.javapoet.ClassName;

public class GrpcClassNames {
    public static final ClassName serviceDescriptor = ClassName.get("io.grpc", "ServiceDescriptor");
    public static final ClassName abstractStub = ClassName.get("io.grpc.stub", "AbstractStub");
    public static final ClassName grpcGenerated = ClassName.get("io.grpc.stub.annotations", "GrpcGenerated");
    public static final ClassName channel = ClassName.get("io.grpc", "Channel");
    public static final ClassName managedChannelLifecycle = ClassName.get("ru.tinkoff.grpc.client", "ManagedChannelLifecycle");
    public static final ClassName grpcClientConfig = ClassName.get("ru.tinkoff.grpc.client.config", "GrpcClientConfig");
}

File: grpc/grpc-client-symbol-processor/src/main/kotlin/ru.tinkoff.grpc.client.ksp/GrpcClassNames.kt
package ru.tinkoff.grpc.client.ksp

import com.squareup.kotlinpoet.ClassName


object GrpcClassNames {
    val serviceDescriptor = ClassName("io.grpc", "ServiceDescriptor")
    val abstractStub = ClassName("io.grpc.stub", "AbstractStub")
    val abstractCoroutineStub = ClassName("io.grpc.kotlin", "AbstractCoroutineStub")
    val grpcGenerated = ClassName("io.grpc.stub.annotations", "GrpcGenerated")
    val channel = ClassName("io.grpc", "Channel")
    val managedChannelLifecycle = ClassName("ru.tinkoff.grpc.client", "ManagedChannelLifecycle")
    val grpcClientConfig = ClassName("ru.tinkoff.grpc.client.config", "GrpcClientConfig")
    val stubFor = ClassName("io.grpc.kotlin", "StubFor")

}
